test/blackbox/common/DDSBlackboxTestsPersistenceGuid.cpp

```diff
@@ -21,6 +21,9 @@
 
 #include <cstring>
 #include <thread>
+#include <sstream>
+#include <iomanip>
+#include <fstream>
```

Comment:
Include what you use: missing \<functional\> header include. Also, sort includes following the style guide.

Reply:
Done in d68b310